### PR TITLE
build mesa for linux

### PR DESCRIPTION
**Pull request by borune-k12**
Specify library name and version:  **lib/1.0**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: .github/workflows/mesa.yml [hunk 1]
```diff
@@ -37,7 +45,7 @@ jobs:
       - name: Install pkg-config
         if: matrix.os == 'macos-latest'
         run: brew install pkg-config
-      - name: Install bison
+      - name: Install biso
```
Comment (mm0d) [inline .github/workflows/mesa.yml:48]: очепятка?

Comment (borune-k12) [reply .github/workflows/mesa.yml]: ага

File: recipes/mesa/all/conanfile.py [hunk 2]
```diff
@@ -194,11 +194,11 @@ def build_requirements(self):
...
-        if self._with_any_opengl:
-            if not tools.which("bison"):
-                self.build_requires("bison_installer/3.3.2@bincrafters/stable")
-            if not tools.which("flex"):
-                self.build_requires("flex_installer/2.6.4@bincrafters/stable")
+        #if self._with_any_opengl:
+        #    if not tools.which("bison"):
+        #        self.build_requires("bison_installer/3.3.2@bincrafters/stable")
+        #    if not tools.which("flex"):
+        #        self.build_requires("flex_installer/2.6.4@bincrafters/stable")
```
Comment (worldemar) [inline recipes/mesa/all/conanfile.py:201]: зачем это в conanfile?

Comment (borune-k12) [reply recipes/mesa/all/conanfile.py:201]: чтоб если мы вдруг когда-то решим не ставить билд тулзы в систему, а брать их из конана, не вспоминать, какие нужны версии

**Pull request merged**